src/utils/src/arg_parser.rs

```diff
@@ -362,7 +362,10 @@ impl<'a> Arguments<'a> {
         // command line arguments by adding just the version argument to the parsed list and
         // returning.
         if args.contains(&VERSION_ARG.to_string()) {
-            let mut version_arg = Argument::new("version").help("Print the binary version number.");
+            let mut version_arg = Argument::new("version").help(
```

Comment:
the `arg_parser` is more general-purpose and is used by multiple binaries in this workspace, that don't have any support for snapshots - jailer and more recently, seccompiler.

We should find another place for this